microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -917,7 +932,7 @@
                 $("#n_archaea_background").text(result["Archaea"].toLocaleString()).addClass("microbe-count").removeClass("spinner-grow spinner-grow-sm");
             });
         retrieveNeighbors(state, 1)
-            .then(queryNeighborAndSelfMetadata(state, ["age_years", "sugary_sweets_frequency"]))
+            .then(queryNeighborAndSelfMetadata(state, ["age_cat", "sugary_sweets_frequency"]))
```

Comment:
If this looks good, then let's keep it, but the corresponding variable would I think be `host_age` followign the metadata updates

Reply:
...same for below

Reply:
host_age doesn't work relative to the current results from microsetta-processing (just tried and reverted). We can poke at that separately.

Reply:
from wgs? maybe lack of samples and okay